Ship_Game/UI/UIButton_Style.cs

```diff
@@ -19,6 +19,7 @@ public enum ButtonStyle
         ResearchQueueUp, // "ResearchMenu/button_queue_up"
         ResearchQueueDown, // "ResearchMenu/button_queue_down"
         ResearchQueueCancel, // "ResearchMenu/button_queue_cancel"
+        ResearchQueueToTop, // "ResearchMenu/button_queue_to_top"
```

Comment:
please add `ResearchQueueToTop` in the last place and not in the middle, since its an enum and loading saves can mess than up if this was saved before that change

Reply:
I though about this, I thought this shouldn't break anything. I thought it just needed to be the correct value representing index in this array:
![image](https://github.com/TeamStarDrive/StarDrive/assets/47790951/7025ea45-eede-41da-b22c-452426151745)
I tested it and it didn't break my old saves (unless it broke something that isn't visible for the player).
Should I move it to the bottom regardless? Imo the code looks better if it's in the same place as the other buttons.

Reply:
yeah, i guess ui styles are not saved. This is a general rule of thump though, as we have a lot of enums which are indeed saved - like DesignRole and DynamicLaunch.
If you see no issues, then no need to change this :)